.github/workflows/ci.yml

```diff
@@ -351,16 +356,14 @@ jobs:
             cd ../../
             git clone https://github.com/libbpf/libbpf.git --branch v1.0.1 --single-branch
             cd libbpf/src && BUILD_STATIC_ONLY=y DESTDIR=/ make install
-            ln -s /usr/lib64/libbpf.a /usr/lib/s390x-linux-gnu/
-
+            ln -s /usr/lib64/libbpf.a /usr/lib/${{env.PLATFORM}}-linux-gnu/
           # Please note: we cannot inject the BPF probe inside QEMU, so right now, we only build it
           run: |
             git config --global --add safe.directory $GITHUB_WORKSPACE
             .github/install-deps.sh
             mkdir -p build
-            cd build && cmake -DBUILD_BPF=On -DUSE_BUNDLED_DEPS=OFF -DMODERN_PROBE_INCLUDE="-I/usr/include/s390x-linux-gnu" -DBUILD_LIBSCAP_MODERN_BPF=ON -DMODERN_BPF_DEBUG_MODE=ON -DBUILD_LIBSCAP_GVISOR=OFF ../
-            KERNELDIR=/lib/modules/$(ls /lib/modules)/build make scap-open driver bpf unit-test-libsinsp -j6
-            ./libsinsp/test/unit-test-libsinsp
+            cd build && cmake -DBUILD_BPF=On -DUSE_BUNDLED_DEPS=OFF -DMODERN_PROBE_INCLUDE="-I/usr/include/${{env.PLATFORM}}-linux-gnu" -DBUILD_LIBSCAP_MODERN_BPF=ON -DMODERN_BPF_DEBUG_MODE=ON -DBUILD_LIBSCAP_GVISOR=OFF ../
```

Comment:
TO avoid issues like: https://github.com/falcosecurity/libs/pull/1522, i'd build `drivers_test` executable instead of `scap-open`.
TO enable its build, add `-DENABLE_DRIVERS_TESTS=On-DCREATE_TEST_TARGETS=On ` to the cmake configure step!

Reply:
Everything else LGTM! Thank you very much!
We are approaching the end :)

Reply:
Thanks a ton for the review and help with the patches. :)